test/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lint-staged-test",
+  "version": "",
+  "description": "",
+  "license": "",
+  "author": "",
+  "type": "module",
+  "engines": {
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0",
+    "npm": ">=8.1.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.0",
+    "@babel/eslint-parser": "7.16.3",
+    "@babel/preset-env": "^7.16.0",
+    "babel-jest": "27.3.1",
+    "consolemock": "^1.1.0",
+    "eslint": "^8.2.0",
```

Comment:
This and other eslint one too?

Reply:
Removed the commit that moved them away.